src/js/gi/components/profile/CalculatorForm.jsx

```diff
@@ -16,7 +16,6 @@ class CalculatorForm extends React.Component {
     this.renderScholarships = this.renderScholarships.bind(this);
     this.renderTuitionAssist = this.renderTuitionAssist.bind(this);
     this.renderEnrolled = this.renderEnrolled.bind(this);
```

Comment:
`onShowModal` is being bound with different arguments for each of those render functions, since it's showing a different modal depending on the section. Each of those would have be set up in the constructor. Will have to revisit along with many other things that need to be refactored.